src/App.js

```diff
@@ -14,56 +19,75 @@ class App extends Component {
+      statusSeeker: null,
     }
 
     this.getKeyWord = this.getKeyWord.bind(this);
   }
 
-  componentDidMount() {     
+  componentDidMount() {
     getWeb3
-    .then(results => {
-      this.setState({
-        web3: results.web3
+      .then(results => {
+        this.setState({
+          web3: results.web3,
+          statusSeeker: statusSeekerContract.withProvider(results.web3.currentProvider)
+        })
+      })
+      .catch((err) => {
+        console.error(err)
       })
-    })
-    .catch(() => {
-      console.log('Error finding web3.')
-    })
   }
 
   getKeyWord(e) {
-    e.preventDefault();
+    e.preventDefault()
+    this.props.dispatch(addKeyword(this.state.statusSeeker))
+  }
 
-    // So we can update state later.
-    var self = this
+  _renderGame = () => {
+    const {
+      currentKeyword,
+      wordList,
+    } = this.props
 
-    // Declare the contract abstractions
-    const contract = require('truffle-contract')
-    const statusSeeker = contract(StatusSeekerContract)
-    statusSeeker.setProvider(this.state.web3.currentProvider)
+    const moveKeyword = (dragIndex, hoverIndex) => {
+      console.log('move keyword')
+      this.props.dispatch(moveKeywordInList(dragIndex, hoverIndex))
+    }
 
-    // Declaring this for later so we can chain functions on StatusSeeker.
-    var statusSeekerInstance
+    return (
+      <div>
+        <div className="button-kw-container">
+          {currentKeyword.isFetching ?
+            <Loading/>
+            : <button className="button-kw" onClick={this.getKeyWord}>Get Key Word</button>}
+        </div>
+        {wordList.length > 0 &&
+          <div>
+            <p>You've found {wordList.length} {wordList.length === 1? 'word' : 'words'}! Look for a total of 12!</p>
+            <KeywordOrganizer keywords={wordList} moveKeyword={moveKeyword} />
+            {wordList.length === 12 &&
+              <div className="button-kw-container copy-button-container">
+                <CopyKeywords keywords={wordList}>Copy Mnemonic</CopyKeywords>
+              </div>
+            }
+          </div>
+        }
+      </div>
+    )
+  }
 
-    // Get accounts.
-    this.state.web3.eth.getAccounts(function(error, accounts) {
-      console.log(accounts)
+  render() {
 
-      statusSeekerInstance = statusSeeker.at("0x9cfd83d56a7937cf7c5afe2281e4738472c4ab61")
-        // Generate random number between 0 and 11. Once we have implemented QR code support
-        // this id will be generated when the users scans the QR and the corresponding word
-        // will be returned without giving away it's position in the array.
-        var id = Math.floor((Math.random() * 11));
+    if (!this.state.web3) {
+      return (<Loading></Loading>)
+    }
 
-        statusSeekerInstance.keyWord.call(id, {from: accounts[0]}).then(function(result) {
-        // Update state with the result.
-        return self.setState({ keyWord: result.toString() })
-      })
-    })
-  }
+    const {
```

Comment:
You can remove this constant now in order to get rid of an unused variable error at compile time.